### PR TITLE
Add task solution

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**
[DEMO LINK](https://SerhiiUnhurian.github.io/react_goods-selector-js/)

### CODE REVIEW

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Unfortunately, your project is not reachable via the demo link(

**Comment by SerhiiUnhurian** [comment]
в мене лінтер не пускає, там треба зробити 2 корективи, але я не знаю як виправити їх, бо коли виправляю, видає нову помилку

Screenshot 2023-11-01 at 12 47 09

Screenshot 2023-11-01 at 12 46 06

**Comment by SerhiiUnhurian** [comment]
задеплоїв)

File: src/App.jsx [hunk 1]
```diff
@@ -14,70 +15,65 @@ export const goods = [
...
+          <h1 className="title is-flex is-align-items-center">
+            {`${selectedGood} is selected`}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
             <button
-              data-cy="AddButton"
+              onClick={() => setSelectedGood('')}
```
Comment (anastasiiavorobiova) [inline src/App.jsx:41]: Consider moving this logic from jsx

```suggestion
              onClick={resetGood}
```

File: src/App.jsx [hunk 2]
```diff
@@ -14,70 +15,65 @@ export const goods = [
...
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
```
Comment (anastasiiavorobiova) [inline src/App.jsx:54]: Consider using the key attribute for rendering list items

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: src/App.jsx (hunk 2 above)
Comment (SerhiiUnhurian) [reply src/App.jsx:58]: but where should i put it here?

Comment (anastasiiavorobiova) [reply src/App.jsx:58]: https://www.developerway.com/posts/react-key-attribute you can find more information here or feel free to ask for help in the chat

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Consider fixing issues from the previous review

File: src/App.jsx (hunk 2 above)
Comment (SerhiiUnhurian) [reply src/App.jsx:58]: seems i did it)

File: src/App.jsx [hunk 3]
```diff
@@ -14,70 +15,71 @@ export const goods = [
...
+          </h1>
+        ) : (
+          <h1 className="title is-flex is-align-items-center">
+            {`${selectedGood} is selected`}
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
             <button
-              data-cy="RemoveButton"
+              onClick={() => resetGood()}
```
Comment (anastasiiavorobiova) [inline src/App.jsx:45]:
```suggestion
              onClick={resetGood}
```

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**